src/core/copyright/copyright_dpkgquery.cpp

```diff
@@ -10,7 +10,13 @@ namespace linuxdeploy {
             using namespace log;
 
             std::vector<fs::path> DpkgQueryCopyrightFilesManager::getCopyrightFilesForPath(const fs::path& path) {
-                subprocess::subprocess proc{{"dpkg-query", "-S", path.c_str()}};
+                fs::path realpath = fs::canonical(path);
+
+                if ((std::string)realpath != (std::string)path) {
+                    ldLog() << LD_INFO << "Canonical path" << realpath << "not equivalent to" << path << std::endl;
```

Comment:
This should be debug level IMO. It's not the application's fault if Debian packages things differently.